Traceback (most recent call last):
  File "/home/jordivilla/miniconda3/lib/python3.10/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/home/jordivilla/miniconda3/lib/python3.10/site-packages/nbclient/client.py", line 1305, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/home/jordivilla/miniconda3/lib/python3.10/site-packages/jupyter_core/utils/__init__.py", line 166, in wrapped
    return loop.run_until_complete(inner)
  File "/home/jordivilla/miniconda3/lib/python3.10/asyncio/base_events.py", line 649, in run_until_complete
    return future.result()
  File "/home/jordivilla/miniconda3/lib/python3.10/site-packages/nbclient/client.py", line 705, in async_execute
    await self.async_execute_cell(
  File "/home/jordivilla/miniconda3/lib/python3.10/site-packages/nbclient/client.py", line 1058, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/home/jordivilla/miniconda3/lib/python3.10/site-packages/nbclient/client.py", line 914, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
import wget

def download_data(dir="~/Data"):
    """
    dir: directory were to store the file
    """
    dir = os.path.expanduser(dir)
    if not os.path.exists(dir):
        os.makedirs(dir)

    site_url = 'ftp://ftp.ebi.ac.uk/pub/databases/intact/current/psi30/pmidMIF30.zip'
    print('\n downloading '+site_url+" into "+dir)
    file_name = wget.download(url=site_url,out=dir)
    print(file_name)

download_data()
------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mModuleNotFoundError[0m                       Traceback (most recent call last)
Cell [0;32mIn[1], line 1[0m
[0;32m----> 1[0m [38;5;28;01mimport[39;00m [38;5;21;01mwget[39;00m
[1;32m      3[0m [38;5;28;01mdef[39;00m [38;5;21mdownload_data[39m([38;5;28mdir[39m[38;5;241m=[39m[38;5;124m"[39m[38;5;124m~/Data[39m[38;5;124m"[39m):
[1;32m      4[0m [38;5;250m    [39m[38;5;124;03m"""[39;00m
[1;32m      5[0m [38;5;124;03m    dir: directory were to store the file[39;00m
[1;32m      6[0m [38;5;124;03m    """[39;00m

[0;31mModuleNotFoundError[0m: No module named 'wget'

